synapse/neuron.py

```diff
@@ -450,35 +450,34 @@ class CryptSeq:
     '''
     Applies and verifies sequence numbers of encrypted messages coming and going
     '''
-    MSGPACK_MAX_INT = 2 ** 63 - 1
     def __init__(self, rx_key, tx_key, initial_rx_seq=0, initial_tx_seq=0):
-        print('CryptSeq: %r %r %r ' % (self, rx_key[:4], tx_key[:4]))
         self._rx_tinh = s_tinfoil.TinFoilHat(rx_key)
         self._tx_tinh = s_tinfoil.TinFoilHat(tx_key)
         self._rx_sn = initial_rx_seq
         self._tx_sn = initial_tx_seq
 
     def encrypt(self, mesg):
         rv = self._tx_tinh.enc(s_msgpack.en((self._tx_sn, mesg)))
-        print('%r sending: %d %r' % (self, self._tx_sn, mesg))
         self._tx_sn += 1
         return rv
 
     def decrypt(self, ciphertext):
         plaintext = self._rx_tinh.dec(ciphertext)
         if plaintext is None:
-            raise s_common.CryptoErr(mesg='Message decryption failure')
+            logger.error(mesg='Message decryption failure')
+            raise s_exc.CryptoErr()
         sn, mesg = s_msgpack.un(plaintext)
-        import pdb; pdb.set_trace()
-        print('%r Got: %d %r' % (self, sn, mesg))
+        # import pdb; pdb.set_trace()
         if sn != self._rx_sn:
-            print('Exception on %r : %d %r' % (self, sn, mesg))
-            raise Exception('Message out of sequence: got %d expected %d' % (sn, self._rx_sn))
+            logger.error('Message out of sequence: got %d expected %d', sn, self._rx_sn)
+            raise s_exc.CryptoErr()
         self._rx_sn += 1
-        print('%r new rx_sn = ', self, self._rx_sn)
         return mesg
 
 
+class ProtoErr(s_exc.SynErr):
```

Comment:
We'll need to move this to synapse.exc. I thought I had removed s_exc import from neuron.py; we should be bringing those exceptions in via s_common